src/storage/ob_i_store.h

```diff
@@ -1573,6 +1573,83 @@ struct ObRowStat {
   TO_STRING_KV(K_(base_row_count), K_(inc_row_count), K_(merge_row_count), K_(result_row_count), K_(filt_del_count));
 };
 
+template <typename T, int64_t LOCAL_BUFFER_SIZE=64>
+class ObRowBuffer {
```

Comment:
ObRowBuffer is not a good name, may be ObDynamicBuffer or..?

Reply:
changed to ObDynamicBuffer